lib/services/api/session_manager.ts

```diff
@@ -118,16 +118,13 @@ export class SNSessionManager extends PureService<SessionEvent> {
   }
 
   public getUser() {
-    if (this.user && !this.apiService.getSession()) {
-      this.alertService.alert(
-        ErrorAlertStrings.MissingSessionBody,
-        ErrorAlertStrings.MissingSessionTitle
-      );
-      SNLog.error(Error('User is defined but no session is present.'))
-    }
```

Comment:
We don't need this anymore as the new provisions in migrations should preclude this use case. This was also bad design because we're presenting an alert inside a getter.